gpii/node_modules/preferencesServer/test/preferencesServiceTests.js

```diff
@@ -0,0 +1,437 @@
+/*!
+Copyright 2017-2018 OCAD university
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+"use strict";
+
+var fluid = require("infusion"),
+    jqUnit = fluid.require("node-jqunit", require, "jqUnit"),
+    gpii = fluid.registerNamespace("gpii");
+
+require("./preferencesServerTestsUtils.js");
+require("./preferencesServerTests.js");
+
+fluid.require("%gpii-universal");
+gpii.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.preferencesServer.preferencesService");
```

Comment:
This file converts the preferencesService tests from in-browser tests to node tests, but I don't see new tests for new APIs like cloudSafeCredCreate() and cloudSafeUnlock().

Reply:
Thanks for catching this.  I added tests for the new endpoints that use these methods, but not for direct usage of the methods.